### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by RyanFriedman36**
Created a navbar component and used routing inside app.js to route to the different pages on the navbar

### CODE REVIEW

File: .vs/ProjectSettings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```
Comment (csjoblinksreddit) [inline .vs/ProjectSettings.json:3]: Im not sure that the .vs files need to be added to the project. Can you add them to the .gitignore file so they are not pushed to github

File: .vs/VSWorkspaceState.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "ExpandedNodes": [
+    "",
+    "\\src"
+  ],
+  "SelectedNode": "\\package.json",
+  "PreviewInSolutionExplorer": false
```
Comment (csjoblinksreddit) [inline .vs/VSWorkspaceState.json:7]: Same comment here. I think if you add the .vs folder it should take care of all of them.

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,54 @@
...
+import { Listings } from './Listings';
+import { NoMatch } from './NoMatch';
+
+function App(props) {
+  const isLoggedIn = props.isLoggedIn
+  if (isLoggedIn){
+    return (
+      <React.Fragment>
+        <Router>
+          <NavigationBar2 />
```
Comment (csjoblinksreddit) [inline src/App.js:27]: I remember you saying that conditional rendering was an issue. Can we walk through why 2 navigation bar components were created?

File: src/About.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styled from 'styled-components';
+const GridWrapper = styled.div`
+  display: grid;
```
Comment (csjoblinksreddit) [inline src/About.js:4]: Can we add the css to a .css file and then import it in? I fear that if we continue down this path our files might begin to get bloated.

File: src/NoMatch.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import styled from 'styled-components';
+const Wrapper = styled.div`
+  margin-top: 1em;
+  margin-left: 6em;
+  margin-right: 6em;
+`;
+export const NoMatch = () => (
+  <Wrapper>
+    <h2>No Match</h2>
```
Comment (csjoblinksreddit) [inline src/NoMatch.js:10]: Can you give me some details for this component?

File: src/components/NavigationBar2.js [hunk 6]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```
Comment (csjoblinksreddit) [inline src/components/NavigationBar2.js:1]: Im not sure that navigationbar2 would be a good name for the different navigation bar components. When we walk through the other navbar can we talk about this too?

File: src/App.js (hunk 3 above)
Comment (RyanFriedman36) [reply src/App.js:27]: When I was reading through how to conditionally render the navbar, it seemed simpler to create one navbar for being logged in and another for being logged out. I can look into changing the level of the condition to the actual links within the navbar, just seemed a bit more involved.

File: src/About.js (hunk 4 above)
Comment (RyanFriedman36) [reply src/About.js:4]: we dont even need that css stuff I can get rid of it.

Comment (RyanFriedman36) [reply src/About.js:4]: yes.

File: src/NoMatch.js (hunk 5 above)
Comment (RyanFriedman36) [reply src/NoMatch.js:10]: This was given in the tutorial for creating the navbar, I do not believe we need it, I will remove it for now.

File: src/components/NavigationBar2.js (hunk 6 above)
Comment (RyanFriedman36) [reply src/components/NavigationBar2.js:1]: Yes. It is logged in vs logged out, I am changing the naming for now but if we decide to bring the conditional down to the level of the links in the nav bar thats okay too, we should just have one navbar.

**Comment by RyanFriedman36** [comment]
Moving to a new pulll request on this branch.

**Pull request closed without merging**